plane/source/docs/soaring.rst

```diff
@@ -205,7 +208,9 @@ Limit maximum distance from home
    Available in firmware revisions 4.1 and later.
 
 
-If using FBWB or CRUISE mode, the parameter :ref:`SOAR_MAX_RADIUS<SOAR_MAX_RADIUS>` can be used to trigger RTL if the aircraft is more than this distance from home when thernalling in LOITER mode. Note that this parameter won't stop the aircraft from exceeding this distance before it enters LOITER mode.
+If using FBWB or CRUISE mode, the parameter :ref:`SOAR_ALT_CUTOFF<SOAR_ALT_CUTOFF>` can be used to trigger RTL if the aircraft is more than this distance from home when thernalling while in LOITER mode. Note that this parameter won't stop the aircraft from exceeding this distance before it enters LOITER mode.
```

Comment:
Incorrect, SOAR_MAX_RADIUS is the relevant parameter - we are talking horizontal distance from home. Pre-existing typo "thernalling" here too, my fault.

Reply:
fixed